pkg/controllermanager/controller/shoot/shoot_conditions_control.go

```diff
@@ -141,3 +134,23 @@ func FilterSeedForShootConditions(obj, oldObj, _ client.Object, _ bool) bool {
 
 	return false
 }
+
+// MergeSeedConditions merges the conditions from the seed in the shoot avoiding duplications with the shoot and seed SystemComponentsHealthy
```

Comment:
```suggestion
// AddSeedConditionsToShoot adds the conditions from the seed to the shoot by adapting the type of the seed's SystemComponentsHealthy condition so that it is not duplicated.
```